### DISCUSSION TITLE
滤镜/着色器小讨论

### DISCUSSION THREAD

**Discussion by diana7127** [diana7127/mpv.net-DW#14] [General]
大家平常看剧看动漫会选择什么滤镜和着色器呢？

**Comment by Vulcankta** [comment]
預設組裡面的A4K

**Comment by Wind-stormger** [comment]
请问有没有横向测评的博文或是帖子，视频，来对比一下所有A4K版本及其他滤镜/着色器？

**Reply by bshht** [reply]
同问

**Comment by bshht** [comment]
SVP的补到60帧似乎不见了？

**Reply by Vulcankta** [reply]
![mpvnet_7q9KtjTaav](https://github.com/diana7127/mpv.net-DW/assets/61764871/ba0f9b4f-825d-40c8-b26c-4dd322a5801e)

**Reply by Wind-stormger** [reply]
再点下面的 强制帧数 就好了
